terraform/environments/mlra/codebuild.tf

```diff
@@ -1,6 +1,8 @@
+# TODO change the module name to codebuild when rebuilding
```

Comment:
Module now includes CodeBuild job to build application, as part of https://dsdmoj.atlassian.net/browse/LAWS-3075